### DISCUSSION TITLE
Does this use yolov5 predictions at every frame?

### DISCUSSION THREAD

**Discussion by Ammar-hala** [mikel-brostrom/yolo_tracking#663] [Q&A]
Hello, 

I'm using yolov5 for real time object detection and trying to find ways through which I can improve detection for my tasks. I trained a yolov5 model and used that model to test on few videos using just yolov5 and also yolov5 with strongsort to see if I can find any differences among them but so far output videos from both looks exactly the same.

I wanted to ask does this code use object detection predictions at every frame? and is there any way I can adjust to use predictions from yolo after specific frames and just use tracking for all remaining frames?

**Comment by mikel-brostrom** [comment]
> I wanted to ask does this code use object detection predictions at every frame?

Yes

> and is there any way I can adjust to use predictions from yolo after specific frames and just use tracking for all remaining frames?

Not out of the box. I would recommend that you cut the video if possible

**Reply by Ammar-hala** [reply]
I don't understand, how would cutting the video help me?
